lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -900,7 +900,19 @@ Context::convertFunction(const slang::ast::SubroutineSymbol &subroutine) {
                  lowering->op.getFunctionType().getInputs())) {
     auto loc = convertLocation(astArg->location);
     auto blockArg = block.addArgument(type, loc);
-    valueSymbols.insert(astArg, blockArg);
+
```

Comment:
If you'd like to get rid of unneeded variables, you could create a list of shadow variables while you build them up:
```suggestion

SmallVector<moore::VariableOp> argVariables;
```